docker-compose.prod.yml

```diff
@@ -21,10 +21,20 @@ services:
     depends_on:
       - server
       - webapp
+    deploy:
+      restart_policy:
+        condition: on-failure
```

Comment:
`on-failure` restarts a container if it was exited with a non-zero exit code, but we want it to be restarted in all cases except if it stopped from the user. I think the `unless-stopped` condition would more appropriate for this.